app/templates/identity.hbs

```diff
@@ -26,30 +27,43 @@
                 {{/if}}
                 Save
             </button> 
+            {{#if (eq this.profileURL this.model.profileURL)}}
+            <BsTooltip @title='This URL is already in use.' @placement='right' />
+            {{else if this.saveDisabled}}
+            <BsTooltip @title='Profile URL is empty!' @placement='right' />
+            {{/if}}
+            </div>
         </div>
 
-        <div class="chaincode-div">
-            <label class="identity-label">
-                Generate Chaincode
-                {{#if this.isChaincodeClicked }}
-                    <label class="chaincode">{{this.Chaincode}}</label>
+    <div class='chaincode-div'>
+      <label class='identity-label'>
+        Generate Chaincode
+        {{#if this.isChaincodeClicked}}
+          <label class='chaincode'>{{this.Chaincode}}</label>
```

Comment:
Can we please fix/revert these quote changes?

Reply:
fixed the quote changes